pulsar-client-cpp/tests/standalone.conf

```diff
@@ -0,0 +1,254 @@
+#
```

Comment:
We want to leave the standalone port as it is since it might be running real workload.